### DISCUSSION TITLE
Высота строк

### DISCUSSION THREAD

**Discussion by tvideo** [totumonline/totum-ru-issues-and-discussions#370] [Есть идея, а что если...]
Здравствуйте. Подскажите, пожалуйста, со следующей задачей:
Расчетная в цикле. Есть ячейка в Колонке, тип Текст или Данные, и в ней есть данные из 10 строк. Можно ли сделать высоту строки, что бы были видны все 10 строчек. Спасибо

**Comment by ttm-alexeypolunin** [comment]
На текущий момент нельзя увеличить высоту строки в строчной части. Заготовки в коде под это есть, но я думаю это упалет в совсем платный функционал

**Comment by tvideo** [comment]
Понял. Спасибо
